django/forms/utils.py

```diff
@@ -42,19 +45,36 @@ def flatatt(attrs):
     )
 
 
+DEFAULT_TEMPLATE_DEPRECATION_MSG = (
+    """
+The "default.html" templates for forms and formsets will be removed. These were
+proxies to the equivalent "table.html" templates, but the new "div.html"
+templates will be the default from Django 5.0. Transitional renderers are
+provided to allow you to opt-in to the new output style now. See
+https://docs.djangoproject.com/en/%s/releases/4.1/ for more details.
+"""
+    % get_docs_version()
```

Comment:
```suggestion
    'The "default.html" templates for forms and formsets will be removed. These were '
    'proxies to the equivalent "table.html" templates, but the new "div.html" '
    "templates will be the default from Django 5.0. Transitional renderers are "
    "provided to allow you to opt-in to the new output style now. See "
    "https://docs.djangoproject.com/en/%s/releases/4.1/ for more details"
    % get_docs_version()
```

Reply:
You're fighting with Black here. 😃

(incoming)

Reply:
Black approved :heavy_check_mark: :wink: